strawberryfields/backends/fockbackend/tests/test_fock_measurement.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+r"""Unit tests for measurements in the Fock basis"""
+
+import pytest
+from itertools import combinations
+import numpy as np
+
+NUM_REPEATS = 50
+
+class TestFockRepresentation:
+    """Tests that make use of the Fock basis representation."""
+
+    def test_vacuum_measurements(self, setup_backend):
+        """Tests Fock measurement on the vacuum state."""
+
+        for _ in range(NUM_REPEATS):
+            backend = setup_backend(3)
+
+            meas = backend.measure_fock([0,1,2])[0]
+            assert np.all(np.array(meas)==0)
+
+    def test_normalized_conditional_states(self, setup_backend, cutoff, tol):
+        """Tests if the conditional states resulting from Fock measurements in a subset of modes are normalized."""
```

Comment:
I'm a bit confused, looks like no measurements are happening in this test?

Reply:
you're right. I changed the test